packages/amplify_core/lib/src/plugin/amplify_api_plugin_interface.dart

```diff
@@ -53,7 +53,7 @@ abstract class APIPluginInterface extends AmplifyPluginInterface {
   void registerAuthProvider(APIAuthProvider authProvider);
 
   // ====== RestAPI ======
-  CancelableOperation<AWSStreamedHttpResponse> delete(
```

Comment:
Realizing that GraphQL operations should have their own type too probably, extending from AWSOperation @ragingsquirrel3 but that can probably be in a separated PR